modules/lvm2/udiskslinuxlogicalvolumeobject.c

```diff
@@ -304,6 +312,20 @@ udisks_linux_logical_volume_object_update (UDisksLinuxLogicalVolumeObject *objec
                                       object->volume_group,
                                       lv_info, meta_lv_info,
                                       needs_polling_ret);
+
+  if (vdo_info)
+    {
+      if (object->iface_vdo_volume == NULL)
+        {
+          object->iface_vdo_volume = udisks_linux_vdo_volume_new ();
+          g_dbus_object_skeleton_add_interface (G_DBUS_OBJECT_SKELETON (object),
+                                                G_DBUS_INTERFACE_SKELETON (object->iface_vdo_volume));
+        }
+      udisks_linux_vdo_volume_update (UDISKS_LINUX_VDO_VOLUME (object->iface_vdo_volume),
+                                                               object->volume_group,
```

Comment:
indentation should be on the first opening parenthesis :-)

Reply:
Fixed.